### ISSUE TITLE
[button] tabindex is not set correctly for disabled button in IE11

### ISSUE THREAD

**Issue by bashmish** [ing-bank/lion#97]
This test currently fails in IE11

```js
it('disabled does not override user provided tabindex', async () => {
  const el = await fixture(`<lion-button tabindex="5" disabled>foo</lion-button>`);
  expect(el.getAttribute('tabindex')).to.equal('-1');
  el.disabled = false;
  await el.updateComplete;
  expect(el.getAttribute('tabindex')).to.equal('5');
});
```

**Comment by bashmish** [comment]
I made some investigation and could not identify the root cause.

One of the symptoms is that our button does not have a getter/setter for `tabindex` in IE11 (`querySelector('lion-button').__lookupGetter__/__lookupSetter__('tabindex')`), while it has it in other browsers. Lack of this breaks syncing between the attribute `tabindex` and the property `tabindex`. Using `tabIndex` (with capital "I") does not solve it either, I even tried to support both, `tabIndex` by default and then create extra getter/setter for `tabindex`, also didn't help. Magic :(

I started working on a simple repro here, but due to a lot of timeouts and instability of StackBlitz today I can't finish it, so for now this repro is WIP: https://stackblitz.com/edit/lit-element-ie11-tabindex-bug?file=index.js
